api/v1alpha2/types.go

```diff
@@ -94,6 +94,11 @@ type NetworkDeviceSpec struct {
 	// will be connected.
 	NetworkName string `json:"networkName"`
 
+	// setName can be used to give the device a more specific/desirable/nicer
+	// name than the default from udev’s ifnames
+	// +optional
+	SetName string `json:"setName,omitempty"`
```

Comment:
Hi @MnrGreg,

The GoDoc comment should reflect the name/casing of the field, not the JSON property. So the comment *should* be:

```golang
// SetName can...
```

And if I might recommend?

```golang
// SetName may be used to explicitly assign a name to the network device
// as it exists in the guest operating system.
```